catalog/app/containers/Bucket/PackageCopyDialog.js

```diff
@@ -189,17 +241,27 @@ function DialogForm({
                 margin="normal"
               />
 
-              <RF.Field
-                component={FilesInput}
-                name="files"
-                validate={validators.nonEmpty}
-                validateFields={['files']}
-                errors={{
-                  nonEmpty: 'Add files to create a package',
-                }}
-                isEqual={R.equals}
-                initialValue={initialFiles}
-              />
+              <div className={classes.filesWrapper}>
+                <RF.Field
+                  className={classes.filesSwitcher}
+                  component={CopyDataSwitcher}
+                  name="copyData"
+                  initialValue
```

Comment:
i believe it should be disabled by default to prevent copying over lots of data unnecessarily